required/amsmath/amsmath.dtx

```diff
@@ -81,11 +81,11 @@ Bug reports can be opened (category \texttt{#1}) at\\%
 %    \begin{macrocode}
 \DeclareRelease{}{1994-06-01}{amsmath-2018-12-01.sty}
 \DeclareRelease{}{2018-12-01}{amsmath-2018-12-01.sty}
-\DeclareCurrentRelease{}{2024-10-30}
+\DeclareCurrentRelease{}{2019-04-01}
 %    \end{macrocode}
 %
 %    \begin{macrocode}
-\ProvidesPackage{amsmath}[2024/10/30 v2.17s AMS math features]
+\ProvidesPackage{amsmath}[2024/10/05 v2.17q AMS math features]
```

Comment:
I still think we need to go _up_ here, so
```suggestion
\ProvidesPackage{amsmath}[2024/11/05 v2.17t AMS math features]
```

Reply:
yes, we have to ... the version is already on ctan and it is a code change (even if it is a reversal)